packages/angular_devkit/build_angular/src/builders/server/index.ts

```diff
@@ -55,6 +56,8 @@ export function execute(
   // Check Angular version.
   assertCompatibleAngularVersion(root);
 
+  void purgeStaleBuildCache(context);
```

Comment:
Is the `void` in order to suppress an unused `Promise` result? Is there perhaps an ignore comment we could use that would make this more clear? I can't say I've ever seen the `void` operator used by a human, so this was quite confusing at first glance.

Outside the style perspective, ignoring a `Promise` always makes me a bit uncomfortable since it's basically racing the entire program and errors don't propagate in a useful way. I get that we don't want to block a user's build on this, but I'm wondering if we should `await` it at the very end so we know it completed successfully without negatively impacting the DX. I assume purging the stale cache is probably quicker than a typical build and only happens when updating the CLI. Otherwise, I'm ok with ignoring the result, just trying to think if there's a way to avoid that pattern without hurting users.

Reply:
I think in this case I’d favor to await the the promise

Reply:
Actually all these methods are not `async`